app/client/src/widgets/ImageWidget/component/index.tsx

```diff
@@ -119,6 +126,14 @@ class ImageComponent extends React.Component<
       cursor = isZoomingIn ? "zoom-in" : "zoom-out";
     }
     if (this.props.onClick) cursor = "pointer";
+
+    if (imageUrl && imageError)
+      return (
+        <ErrorContainer data-testid="error-container">
+          Image load error
```

Comment:
Nit pick: Can we please improve this error message? Also, is it possible to move this to `messages.ts` and use `createMessage` function here?

```suggestion
          Unable to display the image
```